apps/order/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from django.db.models import F
+
+from apps.order.models import CartItem
+from config.serializers import CreatedUpdatedBaseSerializer
+
+
+class CartItemSerializer(CreatedUpdatedBaseSerializer, serializers.ModelSerializer):
+
+    class Meta:
+        model = CartItem
+        fields = ('book', 'quantity',)
+
+    def create(self, validated_data):
+        created_by = self.context['request'].user
+        validated_data['created_by'] = created_by
+        book = validated_data.get('book', '')
```

Comment:
Isn't the book required field?

Reply:
For fields that always exists, let's use [] instead of get()

Reply:
done